src/npc.cpp

```diff
@@ -945,8 +945,7 @@ void npc::do_npc_read()
 
     book = book.obtain( *npc_character );
     if( can_read( *book, fail_reasons ) ) {
-        if( get_player_view().sees( pos() ) ) {
-            add_msg( m_info, _( "%s starts reading." ), disp_name() );
+        add_msg_if_player_sees( pos(), _( "%s starts reading." ), disp_name() );
```

Comment:
Actually should have removed the curly bracket below as well but I am unable to include that in my comment